### PR TITLE
Add Docker functionality

### PR DESCRIPTION
**Pull request by elisiano**
as discussed in #225 I'm adding Docker functionality to this repo.

Once one of the repo administrators will setup integrations with the Docker Hub, the `org.label-schema.docker.cmd` label in the `Dockerfile` can be updated with the right tag and the instructions in the README.md can be changed to a simple `docker pull tag/imagename` instead of building the container locally.

### CODE REVIEW

**Comment by amayer5125** [comment]
I will take a look into this and merge if it works.

File: README.md [hunk 1]
```diff
@@ -72,6 +72,16 @@ LOCAL_CONFIG_DIR=/home/bingrewards/etc
 ```
 Windows: Use build in Task Scheduler
 
+## Docker
```
Comment (amayer5125) [inline README.md]: This should probably go under "Running The Script" section. Maybe make 2 sections "Running Natively" and "Docker"

**Comment by elisiano** [comment]
README.md adjusted as requested

**Comment by kenneyhe** [comment]
I had no luck running multiprocess with python:alpine.  So I had to use ubuntu.  If you don't want multiprocess this change is ok.  just FYI ..

**Comment by elisiano** [comment]
A work around that is to use docker-compose with one config file per account. Now I understand it's not as elegant but it gets the job done... in parallel... on alpine.

**Comment by kenneyhe** [comment]
we can talk later after your checkin.  My mpmain wrapper can split xml to individual accounts and send it to docker services.
Function as a service does auto-scaling with block of 5, so your docker change can be below (https://github.com/kenneyhe/BingRewards.git 's mp_faas branch with some of your functionality):

``` bash
...
COPY entry.sh   /bin
ADD https://github.com/openfaas/faas/releases/download/v0.5-alpha/fwatchdog /usr/bin
RUN chmod +x /usr/bin/fwatchdog /bin/entry.sh
ENV fprocess="/bin/entry.sh"
CMD ["fwatchdog"]
``
entry.sh
``` bash
#!bash -x
timestamp=`date +%s%N`
cat - | openssl base64 -d  | openssl aes-256-cbc -d -k "${KEY}" -out config.xml
# for debug env
echo "started"
python ./main.pyc

```

**Comment by amayer5125** [comment]
I will merge this but I will need to look into getting up an official docker repo. I do not have access to automate builds. So for now people will have to build their own.

**Pull request merged**